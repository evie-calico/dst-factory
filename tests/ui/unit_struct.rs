use dst_factory::make_dst_factory;

#[make_dst_factory]
struct UnitStruct;

fn main() {
}
