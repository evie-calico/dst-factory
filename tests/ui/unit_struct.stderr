error: Unit structs are not supported
 --> tests/ui/unit_struct.rs:4:1
  |
4 | struct UnitStruct;
  | ^^^^^^^^^^^^^^^^^^
